api-reference/50 Common/Object Structures/ExportPivotGridProps/customizeCell.md

```diff
@@ -4,16 +4,232 @@ type: function(options)
+        }
+    }
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+
+    import { DxPivotGridModule } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxPivotGridModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxPivotGrid ...
+            @exporting="onExporting">
+            <DxExport
+                :enabled="true"
+            />
+        </DxPivotGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxPivotGrid, DxExport } from 'devextreme-vue/pivot-grid';
+    import { exportPivotGrid } from 'devextreme/excel_exporter';
+    import ExcelJS from 'exceljs';
+    import saveAs from 'file-saver';
+
+    export default {
+        components: {
+            DxPivotGrid,
+            DxExport
+        },
+        methods: {
+            onExporting(e) {
+                const workbook = new ExcelJS.Workbook();
+                const worksheet = workbook.addWorksheet('Companies');
+
+                exportPivotGrid({
+                    component: e.component,
+                    worksheet: worksheet,
+                    topLeftCell: { row: 2, column: 2 },
+                    customizeCell: function(options) {
+                        const { gridCell, excelCell } = options;
+                        
+                        if(gridCell.rowType === 'D') {
+                            excelCell.font = { color: { argb: 'FF0000FF' }, underline: true };
+                            excelCell.alignment = { horizontal: 'left' };
+                        }
+                    }
+                }).then(function() {
+                    workbook.xlsx.writeBuffer().then(function(buffer) {
+                        saveAs(new Blob([buffer], { type: "application/octet-stream" }), "Companies.xlsx");
+                    });
+                });
+                e.cancel = true;
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import PivotGrid, { Export } from 'devextreme-react/pivot-grid';
+    import ExcelJS from 'exceljs';
+    import saveAs from 'file-saver';
+    import { exportPivotGrid } from 'devextreme/excel_exporter';
+
+    export default function App() {
+        return (
+            <PivotGrid ...
+                onExporting={onExporting}>
+                <Export enabled={true} />
+            </PivotGrid>
+        );
+
+        function onExporting(e) {
```

Comment:
It is better to define auxiliary functions outside the component function: https://stackoverflow.com/a/46138370